cdk/lib/__snapshots__/cdk-playground.test.ts.snap

```diff
@@ -538,6 +538,7 @@ dpkg -i /cdk-playground/cdk-playground.deb",
             },
           },
         ],
+        "Type": "application",
```

Comment:
`Type` gets added in https://github.com/guardian/cdk/pull/576. This change will result in a destructive replacement, however this is a playground app so there's little harm. If anything, it would be great to see RiffRaff protecting us here during CD as RR should fail the deploy since https://github.com/guardian/riff-raff/pull/623.

Reply:
A bit of a chore, but got there in the end. Target Groups cannot be attached to multiple load balancers, therefore we have to forcefully remove the target group first by overriding the logical id:

```typescript
const { targetGroup } = new GuPlayApp(this, { });
(targetGroup.node.defaultChild as CfnElement).overrideLogicalId("targetgrp");
```

And then going back to the automatic logical id.